### PR TITLE
created employee page mission statement

### PR DESCRIPTION
**Pull request by rousell**
Testing
- Go to the employees page and check for a mission statement

### CODE REVIEW

**Comment by zoeames** [comment]
HOW DO I TEST THIS?

**Comment by zoeames** [comment]
👎

File: employee.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Employees</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <h1 class="about-header">About Us</h1>
+    <p>Marzipan cheesecake cheesecake powder oat cake biscuit. Croissant gummies ice cream sugar plum icing jujubes. Cookie marshmallow candy canes jelly beans donut chocolate bar I love. Topping danish donut jelly beans sugar plum apple pie. Gummi bears candy ice cream. Sugar plum sesame snaps cupcake. Gummies jujubes tiramisu halvah oat cake halvah. Marshmallow sesame snaps biscuit I love caramels I love topping bear claw. Biscuit I love cotton candy sesame snaps I love croissant soufflé sugar plum. Ice cream apple pie dragée jujubes powder dragée cupcake carrot cake.</p>
```
Comment (morecallan) [inline employee.html]: Cheesecake is repeated in this mission statement.

**Review by morecallan — COMMENTED** [review]
Mostly good! One comment to be addressed and employee.html should be titled employees.html and it should be in an html directory.

File: employee.html (hunk 1 above)
Comment (zoeames) [reply employee.html]: don't we want more than one cheesecake between the three of us?

File: main.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.about-header {
```
Comment (zoeames) [inline main.css]: can we rename this to title or header?  then we can reuse it for all headers that need to be centered

**Comment by morecallan** [comment]
🦄 magical, wonderful.
👍

**Review by zoeames — COMMENTED** [review]
👍

**Pull request merged**